tests/test_js_syntax.py

```diff
@@ -1,6 +1,6 @@
 from nose.tools import eq_
 
-from js_helper import _do_test_raw
+from .js_helper import TestCase, _do_test_raw
```

Comment:
Is it a letter before an underscore in the "alphabetical order" we use for imports?